### ISSUE TITLE
Pod seems to be broken

### ISSUE THREAD

**Issue by PiXeL16** [ninjaprox/NVActivityIndicatorView#1]
Hi,
I got an error while running the pod install:

```
Installing NVActivityIndicatorView (1.0)

[!] Error installing NVActivityIndicatorView
[!] /usr/bin/git clone https://github.com/ninjaprox/NVActivityIndicatorView.git /var/folders/x6/j9_t239j1pgbcmnjz3nsps3w0000gn/T/d20150730-9266-d4321u --single-branch --depth 1 --branch v1.0

Cloning into '/var/folders/x6/j9_t239j1pgbcmnjz3nsps3w0000gn/T/d20150730-9266-d4321u'...
warning: Could not find remote branch v1.0 to clone.
fatal: Remote branch v1.0 not found in upstream origin
```

**Comment by esttorhe** [comment]
Looking at the [podspec](https://github.com/ninjaprox/NVActivityIndicatorView/blob/master/NVActivityIndicatorView.podspec) it seems that @ninjaprox set it for `1.1` but pushed `1.0` to CocoaPods trunk and there's no `1.0` tag => https://github.com/ninjaprox/NVActivityIndicatorView/releases

Should be an easy fix (either tag `1.0` or push an update to trunk)

**Comment by ninjaprox** [comment]
Thanks @PiXeL16, @esttorhe.
Fixed. Please try again.
